core/arch/arm/include/mm/generic_ram_layout.h

```diff
@@ -0,0 +1,196 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2018, Linaro Limited
+ */
+
+#ifndef __MM_GENERIC_RAM_LAYOUT_H
```

Comment:
The `__MM_` part here, is this something we always do? I.e, prefix with "subsystem"?